api/Screen.json

```
@@ -111,6 +111,41 @@
             "standard_track": false,
             "deprecated": false
           }
+        },
+        "relative-multi-screen": {
+          "__compat": {
+            "description": "Relative to the <a href='https://developer.mozilla.org/docs/Web/API/Window_Management_API/Multi-screen_origin'>Multi-screen origin</a>",
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "1"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "1",
+                "partial_implementation": true,
+                "notes": "Behavior is inconsistent/unreliable."
```

Is "behavior" a good word here? Sounds very generic to me. Could we instead say "Origin relation" to be a bit more precise what exactly is off here? (if you agree, please update throughout).

```suggestion
                "notes": "Origin relation is inconsistent/unreliable."
```

---

@Elchi3 I think you've got a point, but it feels to me like "Origin relation" is a bit difficult to comprehend. How about "Coordinate origin used is inconsistent/unreliable"?

---

Sure, that works, too! :)

---

Cool, notes updated!